core/src/main/java/com/datastax/oss/driver/internal/core/graal/DefaultFeature.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.internal.core.graal;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption;
+import com.oracle.svm.core.annotate.AutomaticFeature;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigFactory;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Optional;
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeClassInitialization;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+
+/**
+ * A Graal feature which determines which checks for well-known config keys corresponding to
+ * instances to be instantiated via reflection, adding each to the current reflection config.
+ */
+@AutomaticFeature
+final class DefaultFeature implements Feature {
+
+  public Config buildConfig() {
+
+    /* Copied from DefaultDriverConfigLoader, including the DEFAULT_ROOT_PATH value there as the literal
+    "datastax-java-driver" below.  Can't reference the class directly here since that will introduce
+    a requirement to specify it as a build-time dependency (as well as the slf4j logger it creates
+    via static init */
+    return ConfigFactory.defaultOverrides()
+        .withFallback(ConfigFactory.defaultApplication())
+        .withFallback(ConfigFactory.defaultReference(CqlSession.class.getClassLoader()))
```

Comment:
I _think_ it would be safer to use `access.getApplicationClassLoader()` as the right CL to locate both application and reference files.

The reason why we use `CqlSession.class.getClassLoader()` has to do with OSGi class loading issues. Here, we are at build-time, trying to generate a native image; and I bet that the only CL available is `access.getApplicationClassLoader()`.